app/models/application_record.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
   scope :with_locale, -> (locale) { where(locale: locale) }
```

Comment:
Layout/SpaceInLambdaLiteral: Do not use spaces between `->` and opening brace in lambda literals